Directory.Build.props

```diff
@@ -1,14 +1,5 @@
 <!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
 <Project>
-  <PropertyGroup>
-    <!-- Remove this when we update Arcade. -->
-    <ImportNetSdkFromRepoToolset>false</ImportNetSdkFromRepoToolset>
-  </PropertyGroup>
-
-  <PropertyGroup>
```

Comment:
This needs to stay because of CreavteVersion....csproj. This is used so that native assets have non-dev monikers